assets/helpers/abTests/abtest.js

```diff
@@ -115,7 +159,7 @@ function userInTest(audiences: Audiences, mvtId: number, country: IsoCountry) {
   const testMin: number = MVT_MAX * audience.offset;
   const testMax: number = testMin + (MVT_MAX * audience.size);
 
-  return (mvtId > testMin) && (mvtId < testMax);
+  return (mvtId > testMin) && (mvtId < testMax) && userInBreakpoint(audience);
```

Comment:
tiny existing error - 
`return (mvtId > testMin) && (mvtId < testMax) && userInBreakpoint(audience);` should be
`return (mvtId >= testMin) && (mvtId < testMax) && userInBreakpoint(audience);`
as if offset is 0 and size is 1 we would want to include everyone from zero to 999999 inclusive rather than 1 to 999999 inclusive

However maybe whoever did this was intentionally trying to exclude the zero index due to buggy cookies?

Reply:
Good spot, I fixed the test framework here: https://github.com/guardian/support-frontend/commit/8a4a106